spec/spec_helper.rb

```diff
@@ -13,6 +13,8 @@
 
 Sidekiq::Logging.logger = Logger.new('log/sidekiq-test.log')
 
+load "#{Rails.root}/db/seeds.rb"
```

Comment:
This shouldn't be necessary. Running `bundle exec rake db:reset` should do the trick.